### PR TITLE
push versions for both master and rc

### PR DESCRIPTION
**Pull request by ryan-s**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -14,7 +14,7 @@ node {
   
   stage ("Deploy"){
     NAMESPACE = k8s.getNamespaceFromBranch(env.BRANCH_NAME)
-    if (env.BRANCH_NAME in ['rc']) {
+    if (env.BRANCH_NAME in ['rc', 'master']) {
```
Comment (squidlegs) [inline Jenkinsfile]: This is an excellent place to try out dev with no staging.

**Review by squidlegs — COMMENTED** [review]
FYI this will do nothing if you don't also update the kustomization yaml!

File: Jenkinsfile [hunk 2]
```diff
@@ -14,7 +14,7 @@ node {
...
       k8s.updateImageTag(NAMESPACE, docker2.imageTag(), 'gcr.io/cyrus-containers/kiwi', env.BRANCH_NAME)
```
Comment (squidlegs) [inline Jenkinsfile:18]: We won't be deploying to the production NAMESPACE for master so you may need to do some extra logic on this one - you can handle it in k8s-bench or here but I think it makes more sense to handle it here.

**Review by squidlegs — CHANGES_REQUESTED** [review]

File: Jenkinsfile (hunk 1 above)
Comment (ryan-s) [reply Jenkinsfile]: Patched to internal-master in latest push

File: Jenkinsfile (hunk 2 above)
Comment (ryan-s) [reply Jenkinsfile:18]: Patched to internal-master in latest push

**Review by squidlegs — APPROVED** [review]

**Review by squidlegs — CHANGES_REQUESTED** [review]
hit approve on wrong PR sorry!

**Review by squidlegs — APPROVED** [review]

**Pull request merged**